src/components/filter.js

```diff
@@ -81,16 +89,24 @@ const Filter = ({
       <ListItem
         innerDivStyle={style.listItemInner}
         leftIcon={<SettingsIcon style={style.listIcon} />}
-        primaryText={<FormattedMessage {...messages.changeFilters} />}
-      />
+      >
+        <Link
```

Comment:
Needs a slight style tweak: the `<a>` rendered does not fill its parent, so if you click away from the text but within ListItem it does not fire.

We could tweak styling, or also do:

```
<Link to="/courses">{
  ({isActive, location, href, onClick, transition}) => 
    <ListItem
          innerDivStyle={style.listItemInner}
          leftIcon={<SettingsIcon style={style.listIcon} />}
          primaryText={<FormattedMessage {...messages.changeFilters} />}
          onClick={onClick}
        />
}</Link>
```
I don't completely understand the `<Link>` docs here, so probably would need a little test to see what the function args actually are https://react-router.now.sh/Link